src/reference/02-DetailTopics/03-Dependency-Management/04-Proxy-Repositories.md

```diff
@@ -25,10 +25,11 @@ increased speed across a team.
 ![image](files/proxy-cloud-setup.png)
 
 There are many good proxy repository solutions out there, with the big
-three being (in alphabetical order):
+four being (in alphabetical order):
```

Comment:
I am not sure if we should list all these as "the big four" since the market share is clearly not equal among them.